packages/mask/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx

```diff
@@ -210,10 +210,7 @@ const useStyles = makeStyles<StyleProps>()((theme, props) => ({
         width: 535,
         margin: '24px auto',
     },
-    claimAllButton: {
-        background: `${theme.palette.mode === 'light' ? '#111418' : '#EFF3F4'} !important`,
-        color: `${theme.palette.mode === 'light' ? '#FFFFFF' : '#0F1419'} !important`,
```

Comment:
Please revert this change since switch button color fixed at #4955

Reply:
Will do the Twitter button theme following in another PR. The dead code button color is not a good idea.